### PR TITLE
Added Jaeger parameters

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

**Review by dmitri-d — APPROVED** [review]

File: topics/docs/installation/installation-options.adoc [hunk 1]
```diff
@@ -271,17 +271,43 @@ If you intend to use a custom image, you must override all three values of hub,
 [options="header"]
 |======
 |Parameter |Description |Default
-|hub
+|`hub`
 |The hub that the Operator uses to pull Jaeger images
 |`jaegertracing/` or `registry.redhat.io/openshift-service-mesh/`
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc:276]: This is a community doc. Does it make sense to mention the product images? The default value is always `jaegertracing`, correct?

File: topics/docs/installation/installation-options.adoc [hunk 2]
```diff
@@ -271,17 +271,43 @@ If you intend to use a custom image, you must override all three values of hub,
...
 |`tag`
 |The tag that the Operator uses to pull the Jaeger images
-|A valid container image tag.
 |`1.13.1`
 |`template`
 |The deployment template to use for Jaeger
 |`all-in-one`/`production-elasticsearch`
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc:282]: Same here. The default value is always `all-in-one`.

File: topics/docs/installation/installation-options.adoc (hunk 1 above)
Comment (brian-avery) [reply topics/docs/installation/installation-options.adoc:276]: Yep. Policy all along has been for both product and community to include references to each other's parameters. Jaegertracing is the default in community, but not product.

File: topics/docs/installation/installation-options.adoc (hunk 2 above)
Comment (brian-avery) [reply topics/docs/installation/installation-options.adoc:282]: all-in-one is the default in community but not product.

**Review by jwendell — APPROVED** [review]

**Pull request merged**